cfme/tests/v2v/test_v2v.py

```diff
@@ -1,15 +1,60 @@
-"""Test to validate basic navigations.Later to be replaced with End-to-End functional testing."""
 import pytest
 
+from cfme.configure.configuration.region_settings import Tag, Category
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.hosts import setup_host_creds
+from cfme.utils import testgen
 
-pytestmark = [pytest.mark.ignore_stream('5.8', '5.9')]
+
+def pytest_generate_tests(metafunc):
+    argnames1, argvalues1, idlist1 = testgen.providers_by_class(metafunc, [VMwareProvider])
```

Comment:
**REQUIRED**: This is no longer the right way to do this. We do not want to iterate over the specific providers. If we need to test against multiple combinations of different types of providers, then we should specify the types and use a resolver to try to resolve them, any missing ones should skip the tests. If you need help in doing this, please let me know.